projects/libsndfile/project.yaml

```diff
@@ -0,0 +1,10 @@
+homepage: "https://github.com/erikd/libsndfile"
+primary_contact: "evpobr@gmail.com"
+language: c
+auto_ccs:
+  - "cmeister2@gmail.com"
+sanitizers:
+  - address
+  - undefined
+  - memory:
+      experimental: true
```

Comment:
We don't recommend doing this, do you see any major issues when running the fuzzer in this config.

Reply:
Running locally I haven't seen any problems. I can remove it if it's going to be a problem?

Reply:
Yes, please remove then. We don't recommend using experimental now.